library/src/androidTest/java/com/jmolsmobile/landscapevideocapture/configuration/CaptureConfigurationBuilderTest.java

```diff
@@ -111,4 +111,37 @@ public void builtConfigurationAllowFrontCamera() throws Exception {
 
         assertThat(configuration.getAllowFrontFacingCamera()).isFalse();
     }
+
+    @Test
+    public void builtConfigurationHideFlashToggleAndFlashStartOFF() throws Exception {
+        CaptureConfiguration configuration = new Builder(MOCK_RESOLUTION, MOCK_QUALITY).setFlashOption(PredefinedCaptureConfigurations.FlashOption.HIDE).build();
+
+        assertThat(configuration.getAllowFlashToggle()).isFalse();
+        assertThat(configuration.getIfFlashStartOn()).isFalse();
```

Comment:
Please split each and every one of those tests in two. Every test should only have exactly one assertion/verification.

Reason is that with multiple asserts: tests can fail for more than one reason. So if they fail you need to actually do an investigation to see what went wrong. With only one assert the root cause for failure is crystal clear (especially if you gave the test a great name). Second benefit is that the tests stops running after the first failure which could be annoying if you actually fixed bug number 1, but there is still a second one.